pet-finder/src/Components/Forms/LoginForm/index.tsx

```diff
@@ -16,28 +24,48 @@ export const LoginForm: FC<LoginFormProps> = () => {
     register,
     handleSubmit,
     formState: { errors, isValid },
-  } = useForm({ mode: 'all', resolver: yupResolver(loginSchema()) })
-  const onSubmit: SubmitHandler<LoginFormType> = ({ email, password }) => {
-    console.log(email, password)
-  }
+    setError,
+  } = useForm({
+    mode: 'all',
+    resolver: yupResolver(loginSchema(ERROR_MESSAGES)),
+  })
+
+  const onSubmit: SubmitHandler<LoginFormType> = useCallback(
+    async ({ email, password }) => {
+      console.log(email, password)
```

Comment:
no console

Reply:
Lets leave this logs, work is not finished here